### PR TITLE
feat: added publish guidelines for pkgjs owned packages

### PR DESCRIPTION
**Pull request by wesleytodd**
As discussed in https://github.com/nodejs/package-maintenance/issues/410.

### CODE REVIEW

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by dominykas — APPROVED** [review]

File: Governance.md [hunk 1]
```diff
@@ -177,6 +177,14 @@ In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
 of Directors approval may also be required. (Most likely in the case of
 transfeering a repository into or out of the organization).
 
+##### Publishing Packages
+
+When publishing packages owned by the pkgjs org, the maintainers of the project
+can choose to use the global scope or the `@pkgjs` scope on the registry.  When
+the package is first published, we always turn on required 2FA to publish.  In
```
Comment (ljharb) [inline Governance.md:184]:
```suggestion
can choose to use the global scope or the `@pkgjs` scope on the registry. When
the package is first published, we always turn on required 2FA to publish. In
```

**Review by ljharb — APPROVED** [review]

**Review by rodion-arr — APPROVED** [review]

File: Governance.md (hunk 1 above)
Comment (wesleytodd) [reply Governance.md:184]: Ah yes, despite how hard I try....I cannot escape the muscle memory of "two spaces after a sentence" being taught to me my entire education.

File: Governance.md [hunk 2]
```diff
@@ -177,6 +177,14 @@ In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
 of Directors approval may also be required. (Most likely in the case of
 transfeering a repository into or out of the organization).
 
+##### Publishing Packages
+
+When publishing packages owned by the pkgjs org, the maintainers of the project
+can choose to use the global scope or the `@pkgjs` scope on the registry. When
+the package is first published, we always turn on required 2FA to publish. In
+the case of using the global scope, you also need to transfer ownership of the
+package so the `@pkgjs` org on npm.
```
Comment (styfle) [inline Governance.md:186]:
```suggestion
package to the `@pkgjs` org on npm.
```

Is this a typo?

File: Governance.md [hunk 3]
```diff
@@ -177,6 +177,14 @@ In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
 of Directors approval may also be required. (Most likely in the case of
 transfeering a repository into or out of the organization).
 
+##### Publishing Packages
+
+When publishing packages owned by the pkgjs org, the maintainers of the project
+can choose to use the global scope or the `@pkgjs` scope on the registry. When
+the package is first published, we always turn on required 2FA to publish. In
```
Comment (styfle) [inline Governance.md:184]:
```suggestion
the package is first published, we always turn on "required 2FA to publish". In
```

Is this the literal text? How about quotes so its clear what to look for in the UI?

Comment (wesleytodd) [reply Governance.md:184]: The exact text is "Require two-factor authentication to publish".  I like using that, but npm could always change it, so I wouldn't worry too much about it other than stating we turn that feature on.

```suggestion
the package is first published, we always turn on "Require two-factor authentication to publish". In
```

**Review by styfle — APPROVED** [review]

File: Governance.md [hunk 4]
```diff
@@ -177,6 +177,14 @@ In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
 of Directors approval may also be required. (Most likely in the case of
 transfeering a repository into or out of the organization).
 
+##### Publishing Packages
+
+When publishing packages owned by the pkgjs org, the maintainers of the project
```
Comment (Eomm) [inline Governance.md:182]:
```suggestion
When publishing packages owned by the [pkgjs](https://github.com/pkgjs) org, the maintainers of the project
```

**Review by Eomm — APPROVED** [review]
Suggestion added with a link to the org

**Pull request merged**